recipe/meta.yaml

```diff
@@ -1,16 +1,16 @@
-{% set version = "0.12" %}
+{% set version = "0.13" %}
 
 package:
   name: patchelf
   version: {{ version }}
 
 source:
   url: https://github.com/NixOS/patchelf/archive/{{ version }}.tar.gz
-  sha256: 3dca33fb862213b3541350e1da262249959595903f559eae0fbc68966e9c3f56
+  sha256: 60c6aeadb673de9cc1838b630c81f61e31c501de324ef7f1e8094a2431197d09
   patches:
     # Fix unsupported overlap of SHT_NOTE and PT_NOTE
     # See https://github.com/NixOS/patchelf/pull/230
-    - 6edec83653ce1b5fc201ff6db93b966394766814.patch
+#    - 6edec83653ce1b5fc201ff6db93b966394766814.patch
```

Comment:
For clarity, I would:
- completely delete these three lines (the two comments plus the patch list item);
- remove the `.patch` file from the repository using `git rm`;
- commit with a (commit) message like:
```
Removed 6edec8365....patch

This patch has been merged upstream into the 0.13 release;
see https://github.com/NixOS/patchelf/pull/230.
```